Procfile

```diff
@@ -1 +1,4 @@
-release: yarn backend sequelize-cli db:migrate && yarn backend sequelize-cli db:seed:all 
```

Comment:
This file is currently used by our Heimdall Server Heroku deployments, removing the release command will break our demo environments.

Reply:
cloud foundry also uses a procfile https://docs.cloudfoundry.org/buildpacks/prod-server.html , could the `release` be added back to resolve or is that single command the only command allowed for the file? I can setup a cloud foundry specific bash script and cfignore the procfile as an alternative here

Reply:
I believe Heroku will ignore commands that it does not recognize, so just adding the `release` command back should be fine.